paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -192,118 +192,6 @@
                 "autoscaling": {
                     "type": "object"
                 },
-                "horizontal_autoscaling": {
-                    "type": "object",
-                    "additionalProperties": false,
-                    "properties": {
```

Comment:
It looks like there's one instance of compute-infra-test-service in norcal-stagef still using this section, so you probably need to change/remove it before shipping this